Sources/Onboarding/ProductNotifications/CookieNotificationBlockingConsentView.swift

```diff
@@ -22,20 +23,20 @@ public struct CookieNotificationBlockingConsentView: View {
   
   @Environment(\.presentationMode) @Binding private var presentationMode
   @State private var showAnimation = false
-  
-  public var onYesButtonPressed: (() -> Void)?
+  public let onYesButtonPressed: YesCallback
 
   private var yesButton: some View {
-    
     Button(Strings.yesBlockCookieConsentNotices,
       action: {
         withAnimation(animation) {
           self.showAnimation = true
         }
       
         Task { @MainActor in
+          try await Task.sleep(seconds: 3.5)
```

Comment:
Maybe better to put sleep after? so the user does not dismiss the view or something

Reply:
Done!